src/uharfbuzz/_harfbuzz.pyx

```diff
@@ -379,6 +392,18 @@ cdef class Face:
     def upem(self, value: int):
         hb_face_set_upem(self._hb_face, value)
 
+    @property
+    def reference_blob(self) -> Blob:
+        cdef hb_blob_t* blob = hb_face_reference_blob(self._hb_face)
+        if blob is NULL:
+            raise MemoryError()
+        cdef Blob inst = Blob(None)
+        inst._hb_blob = blob
+        cdef unsigned int blob_length
+        cdef const_char* blob_data = hb_blob_get_data(blob, &blob_length)
+        inst._data = blob_data[:blob_length]
```

Comment:
Why is this done here and not in `Blob.data()`? may be also add `Blob.from_ptr()` and simplify the rest of this?

Reply:
I agree.